addons/website_event_sale/views/website_event_templates.xml

```diff
@@ -23,6 +23,9 @@
             <span t-esc="event.event_ticket_ids[-1].price_reduce" t-options="{'widget': 'monetary', 'display_currency': website.pricelist_id.currency_id}"/>
         </span>
     </xpath>
+    <xpath expr="//div[hasclass('o_wevent_price_range')]" position="attributes">
+        <attribute name="class" add="px-2" separator=" " />
+    </xpath>
```

Comment:
Not sure I understand this "px-2" addition.

The only time it's applying is when the element is hidden:
`<div id="price-range" class="o_wevent_price_range d-none px-2">`

When you enable the "Fold ticket details", the override removes the "px-2" class:
`<div id="price-range" class="o_wevent_price_range">`

So it really looks like this is unused anyway?
Can you double-check and remove this if it's the case?